hw_20/bank.py

```diff
@@ -0,0 +1,50 @@
+"""Homework_11"""
+
+# # # Банковский вклад # # #
+# Создайте класс вклад, который содержит необходимые поля и методы,
+# например сумма вклада и его срок.
+# Пользователь делает вклад в размере N рублей сроком на R лет под 10% годовых
+# (вклад с возможностью ежемесячной капитализации - это означает,
+# что проценты прибавляются к сумме вклада ежемесячно).
+# Написать класс Bank, метод deposit принимает аргументы N и R,
+# и возвращает сумму, которая будет на счету пользователя.
+#
+# https://myfin.by/wiki/term/kapitalizaciya-procentov
+
+
+class Deposit:
+    """Определение класса Deposit"""
+    def __init__(self, amount, term, percent=0.10):
+        self.amount = amount
+        self.term = term
+        self.percent = percent
+
+# Расчет при ежемесячной капитализации:
+    # S = Р * (1 + (N / 100) / 12) ^ M
+        # S – сумма вклада, которую вкладчик получит на руки;
+        # Р – изначально внесенная сумма
+        # N - %ставка(годовая)
+        # срок вклада в месяцах
+    # Пример
+        # Сумма вклада – 50 000 руб.;
+        # Годовая ставка - 8 %;
+        # Срок вклада – 12 мес.
+        # 50000х(1 + 0, 08 / 12) ^ 12 = 54150руб.
+
+    def total_amount(self):
+        return self.amount * ((1 + self.percent/12)**self.term)
+
+
+class Bank:
+    """Определение класса Bank"""
+    def deposit(self, N, R):
+        deposit = Deposit(N, R)
+        return deposit.total_amount()
+
+
+bank = Bank()
+N = 1000  # сумма вклада
+R = 12  # срок вклада в месяцах
+total_amount = bank.deposit(N, R)
+
+print(f"Сумма на счету через {R} месяцев: {total_amount:.4f} рублей")
```

Comment:
if you do not want to execute some code when module is imported, use `if __name__ == "__main__"`

Reply:
done